submissions/kolawoleOgunfowokan/HTML/index.html

```diff
@@ -197,37 +197,149 @@
             </div>
         </nav>
     </header>
-    <section class="products">
+    <section class="products"  class="product-box">
```

Comment:
Please merge class attributes

Reply:
@taiwo-ladipo This has been resolved. Thank you.